src/lib/crypto/kimchi_bindings/stubs/rust-toolchain.toml

```diff
@@ -11,4 +11,4 @@
 # 4. figure out the hashes of the (now obsolete) docker images used in CI rules that are failing, grep for these hashes and replace them with the new hashes
 
 [toolchain]
-channel = "1.72"
+channel = "1.79.0"
```

Comment:
Do we want to force `.0` specifically? Why not leave the minor version open?

Reply:
I do not want specifically `.0`. It is mostly because I went on the [release of Rust 1.79.0](https://releases.rs/docs/1.79.0/) to get a release date for the nightly version.